### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by devahmadmohammadi** [s3bubble/Streamium-Netflix-Theme#85]
I would like to have a landing page on my website and there is no option for that

**Comment by jdphiladelphia** [comment]
@devahmadmohammadi , FWIW, I created a landing page on my Streamium Theme website. Just install the plugin "Blank Slate," apply that to a page in your site and build your landing page.

If are referring to a homepage people "land on" before heading over to your listing of videos, then setup the new page you create to be the FrontPage under Settings > Reading > Homepage. Simple as that (or at least, it worked for me simply).

Hope this helps!

**Comment by realwildgirls** [comment]
Yes that works, but it doesn't load the index page correctly with video slider/loading series and movies category or the continue watching row. 
If that all works correctly for you and you have a landing page set up let me know..thanks

Terry

**Comment by newibn** [comment]
jdphiladelphia I installed the plugin but I don't see how to customize it. Can you please assist?

**Comment by newibn** [comment]
Never mind I got it. Thanks anyway.
